src/modules/menu/components/share/MenuShareEmbed.vue

```diff
@@ -295,7 +299,22 @@ watch(
                 </div>
                 <!-- eslint-disable vue/no-v-html-->
                 <div class="small text-wrap text-center" v-html="t('share_disclaimer')"></div>
-                <!-- eslint-enable vue/no-v-html-->
```

Comment:
the re-enabling of the rule should still be added, it otherwise disable this rule for the remainder of the file